app/database_etl/postgres_tables_handler/table_generator.py

```diff
@@ -76,11 +76,26 @@ def create_multi_select_tables(original_data, current_time):
         # Create dataframe for table with an id column, a name column, and a created_at column
         new_df_cols = [id_col, name_col, 'created_at']
         col_specific_df = pd.DataFrame(columns=new_df_cols)
+
+        # Note: We are appending country iso2 to each
+        # city/state temporarily because we must create a country iso2 column
+        if col == "state" or col == "city":
+            original_data[col] = original_data.apply(lambda row: add_country_iso2_to_stringlist(row[col], row['country']), axis=1)
```

Comment:
ah - so the string list is a list of states or cities... still change the param name

Reply:
ok